docs/about/all/releases/security-release-notes-202409.0.md

```diff
@@ -87,7 +87,7 @@ composer update spryker-shop/customer-page
 composer show spryker-shop/customer-page # Verify the version
 ```
 
-2. In `src/Pyz/Zed/Product/ProductConfig.php`, add the following method:
+2. In `src/Pyz/Yves/CustomerPage/CustomerPageConfig.php`, add the following method in case you cannot update the `spryker-shop/customer-page` module to version 2.54.0 or higher:
```

Comment:
The `in case you cannot update the /`spryker-shop/customer-page/` module to version 2.54.0 or higher` does not make sense to me: if the module is not of the 2.54.0, the method `isRememberMeEnabled` would not exist in the module and thus the project-level override would not be used. 

The class name change is valid.

Reply:
Changed